Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -198,23 +228,25 @@ public void SolverUpdateEntry()
         /// </remarks>
         /// <param name="position"></param>
         /// <param name="rotation"></param>
-        public virtual void SnapTo(Vector3 position, Quaternion rotation)
+        public virtual void SnapTo(Vector3 position, Quaternion rotation, Vector3 scale)
```

Comment:
It might be good to leave the previous version around, with an [`Obsolete` attribute](https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=netframework-4.8) and guidance to move to the new one, so we don't break any custom solvers that might have overridden the old method signature.

Reply:
Same for `SnapGoalTo` below.